README.md

```diff
@@ -1 +1 @@
-testing semantic versioning 5.0.0
```

Comment:
The code changes appear to update the version of semantic versioning being tested from 5.0.0 to 5.2.0. This indicates a minor version increase, which could potentially introduce new features, improvements, or bug fixes. The reviewer should verify that this change aligns with the project's requirements and that any necessary updates or testing are performed accordingly.